src/OpenTelemetry.Exporter.NewRelic/NewRelicTraceExporter.cs

```diff
@@ -224,7 +214,7 @@ private Span ToNewRelicSpan(Activity openTelemetrySpan)
                 newRelicSpanBuilder.WithServiceName(_config.ServiceName);
             }
 
-            if (openTelemetrySpan.ParentSpanId != default)
+            if (openTelemetrySpan.ParentSpanId != EmptyActivitySpanId)
```

Comment:
This works, but I believe the check for `default` should have worked as well. Not sure yet, but may be a .NET bug. I will be following up on this to confirm.